WordPress/Classes/Stores/ActivityStore.swift

```diff
@@ -59,15 +61,18 @@ private enum ActivityStoreError: Error {
 
 class ActivityStore: QueryStore<ActivityStoreState, ActivityQuery> {
 
-    fileprivate let refreshInterval: TimeInterval = 60 // seconds
+    private let refreshInterval: TimeInterval = 60 // seconds
```

Comment:
Not sure you need the seconds comment there since TimeInterval is inherently defines it as seconds anyway
```suggestion
    private let refreshInterval: TimeInterval = 60
```

Reply:
This comment was already there. I just changed `fileprivate` to `private`, but anyway, removed. :)

Reply:
Ah, I missed that 🤦 . Thanks for changing 👍